ERROR - 2023-03-29 12:15:50 --> mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `emplead...', 0)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `emplead...')
#2 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `emplead...')
#3 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `emplead...', Array, false)
#4 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proyecto\app\Models\EmpleadosModel.php(48): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proyecto\app\Controllers\Empleados.php(32): App\Models\EmpleadosModel->obtenerEmpleados()
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Empleados->index()
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#10 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-29 12:15:50 --> MySQL server has gone away
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `empleados`.*, `municipios`.`nombre` as `nombreMuni`, `cargos`.`nombre` as `nombreCargo`, `departamentos`.`nombre` as `nombre_dpto`, `paises`.`nombres` as `nombrePais`
FROM `empleados`
JOIN `municipios` ON `municipios`.`id` = `empleados`.`id_municipio`
JOIN `departamentos` ON `departamentos`.`id` = `municipios`.`id_dpto`
JOIN `paises` ON `paises`.`id` = `departamentos`.`id_pais`
JOIN `cargos` ON `cargos`.`id` = `empleados`.`id_cargo`
WHERE `empleados`.`estado` = :empleados.estado:
ORDER BY `empleados`.`id`', [...], false)
 2 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\EmpleadosModel.php(48): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Empleados.php(32): App\Models\EmpleadosModel->obtenerEmpleados()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Empleados->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-29 12:16:35 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bd_taller', 3306, '', 0)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proyecto\app\Controllers\Paises.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Paises->index()
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\proyecto\app\Controllers\Paises.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Paises->index()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-29 12:16:35 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `paises`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Paises.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Paises->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-29 12:16:37 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bd_taller', 3306, '', 0)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proyecto\app\Controllers\Paises.php(56): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Paises->eliminados()
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\proyecto\app\Controllers\Paises.php(56): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Paises->eliminados()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-29 12:16:37 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `paises`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Paises.php(56): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Paises->eliminados()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-29 12:16:38 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bd_taller', 3306, '', 0)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proyecto\app\Controllers\Paises.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Paises->index()
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\proyecto\app\Controllers\Paises.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Paises->index()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-29 12:16:38 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `paises`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Paises.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Paises->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-29 12:16:47 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bd_taller', 3306, '', 0)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `emplead...', Array, false)
#4 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proyecto\app\Models\EmpleadosModel.php(48): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proyecto\app\Controllers\Empleados.php(32): App\Models\EmpleadosModel->obtenerEmpleados()
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Empleados->index()
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#10 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `emplead...', Array, false)
#3 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\proyecto\app\Models\EmpleadosModel.php(48): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\proyecto\app\Controllers\Empleados.php(32): App\Models\EmpleadosModel->obtenerEmpleados()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Empleados->index()
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-29 12:16:47 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `empleados`.*, `municipios`.`nombre` as `nombreMuni`, `cargos`.`nombre` as `nombreCargo`, `departamentos`.`nombre` as `nombre_dpto`, `paises`.`nombres` as `nombrePais`
FROM `empleados`
JOIN `municipios` ON `municipios`.`id` = `empleados`.`id_municipio`
JOIN `departamentos` ON `departamentos`.`id` = `municipios`.`id_dpto`
JOIN `paises` ON `paises`.`id` = `departamentos`.`id_pais`
JOIN `cargos` ON `cargos`.`id` = `empleados`.`id_cargo`
WHERE `empleados`.`estado` = :empleados.estado:
ORDER BY `empleados`.`id`', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\EmpleadosModel.php(48): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Empleados.php(32): App\Models\EmpleadosModel->obtenerEmpleados()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Empleados->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Empleados))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-29 12:17:00 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bd_taller', 3306, '', 0)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `departa...', Array, false)
#4 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proyecto\app\Models\DepartamentosModel.php(39): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proyecto\app\Controllers\Departamentos.php(19): App\Models\DepartamentosModel->obtenerDepartamentos()
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Departamentos->index()
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Departamentos))
#10 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `departa...', Array, false)
#3 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\proyecto\app\Models\DepartamentosModel.php(39): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\proyecto\app\Controllers\Departamentos.php(19): App\Models\DepartamentosModel->obtenerDepartamentos()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Departamentos->index()
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Departamentos))
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-29 12:17:00 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `departamentos`.*, `paises`.`nombres` as `nombre_pais`
FROM `departamentos`
JOIN `paises` ON `paises`.`id` = `departamentos`.`id_pais`
WHERE `departamentos`.`estado` = :departamentos.estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\DepartamentosModel.php(39): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Departamentos.php(19): App\Models\DepartamentosModel->obtenerDepartamentos()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Departamentos->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Departamentos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-29 12:17:07 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'bd_taller', 3306, '', 0)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\proyecto\app\Controllers\Paises.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Paises->index()
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\proyecto\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\proyecto\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\proyecto\app\Controllers\Paises.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Paises->index()
#7 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
#8 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-03-29 12:17:07 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `paises`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Paises.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Paises->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Paises))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-29 16:38:27 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `mu...', 0)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mu...')
#2 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mu...')
#3 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mu...', Array, false)
#4 C:\xampp\htdocs\proyecto\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyecto\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyecto\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\proyecto\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\proyecto\app\Controllers\Municipios.php(57): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#10 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#11 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-03-29 16:38:27 --> Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `municipios` (`id_dpto`, `nombre`, `fecha_crea`) VALUES (:id_dpto:, :nombre:, :fecha_crea:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Municipios.php(57): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-29 16:45:21 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\proyecto\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `mu...', 0)
#1 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `mu...')
#2 C:\xampp\htdocs\proyecto\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `mu...')
#3 C:\xampp\htdocs\proyecto\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `mu...', Array, false)
#4 C:\xampp\htdocs\proyecto\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyecto\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyecto\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\proyecto\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\proyecto\app\Controllers\Municipios.php(57): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
#10 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
#11 C:\xampp\htdocs\proyecto\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\proyecto\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-03-29 16:45:21 --> Cannot add or update a child row: a foreign key constraint fails (`bd_taller`.`municipios`, CONSTRAINT `municipios_ibfk_1` FOREIGN KEY (`id_dpto`) REFERENCES `departamentos` (`id`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `municipios` (`id_dpto`, `nombre`, `fecha_crea`) VALUES (:id_dpto:, :nombre:, :fecha_crea:)', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Municipios.php(57): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Municipios->insertar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Municipios))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
